clickhouse_sqlalchemy/engines.py

```diff
@@ -328,3 +347,26 @@ class Memory(_NoParamsEngine):
 
 class Null(_NoParamsEngine):
     pass
+
+
+class File(Engine):
+
+    __visit_name__ = 'file'
```

Comment:
Good job!

It seems there is no need in custom visit name. `__visit_name__ = 'engine'` from `Engine` is enough. `visit_file` also can be dropped.

Reply:
You are right